packages/flutter_tools/templates/create/android.iml.tmpl

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$/android">
+      <sourceFolder url="file://$MODULE_DIR$/android/app/src/main/java" isTestSource="false" />
+    </content>
+    <orderEntry type="jdk" jdkName="Android API 25 Platform" jdkType="Android SDK" />
```

Comment:
Okay, I will add a template variable for the version number.

If the user runs "flutter create" from the command line and opens the project, there is no chance for us to create a new SDK for them. Adding the version number to the name is a good hint as to which one they should attach. (Also, it might exactly match the Android SDK if they kept the default name that IDEA picks.)